indexer/feature.hpp

```diff
@@ -41,12 +40,12 @@ class FeatureBase
 
   feature::EGeomType GetFeatureType() const;
 
-  inline uint8_t GetTypesCount() const
+  uint8_t GetTypesCount() const
```

Comment:
It is not a valid reason to remove inlines in this PR. Please do it separately (or better leave as it is - I don't see any harm in direct instructions to compiler "please try make this method faster").